action.yml

```
@@ -161,6 +161,11 @@ runs:
         labels: ${{inputs.labels}}
         token: ${{inputs.token}}
 
+    - name: Check outputs
+      if: ${{ steps.cpr.outputs.pull-request-number }}
```

I think this might make more sense, right?
```suggestion
      if: ${{ steps.create-pr.outputs.pull-request-url }}
```

---

Weird! Could have sworn I had made that change :man_shrugging: Thanks! Fixed!